packages/fiori-elements-writer/test/__snapshots__/fpm.test.ts.snap

```diff
@@ -3669,7 +3669,7 @@ archive.zip
     \\"@ui5/cli\\": \\"^2.14.1\\",
     \\"@sap/ux-ui5-tooling\\": \\"1\\",
     \\"@sap/ux-specification\\": \\"latest\\",
-    \\"@sapui5/ts-types-esm\\": \\"~1.96.11\\",
+    \\"@sapui5/ts-types-esm\\": \\"~1.96.0\\",
```

Comment:
why is the patch dropped? 
more or less the same....

Reply:
With the latest changes to both types method, if a version is specified that is within the correct range, the patch is dropped, this will ensure the latest of the minor release version is used. There is no guarntee that the patch version supplied will match what is available in `@sapui5/ts-types-esm` or `@sapui5/ts-types` version tree.

Reply:
Good point. Thanks